Migration of Pages from Old Website

jt15s opened the pull request:
As per the [discussions on the forums](https://discuss.haiku-os.org/t/404-error-on-haiku-inc-website/10157/), I am migrating the old pages (on the Wayback Machine) from the Haiku Inc website. I've also updated the copyright notice at the bottom to 2020. If there is anything that needs correcting, feel free to correct it! Thanks!

jt15s committed:
Migration of Pages from Old Website

As per the discussions on the forums, I am migrating the old pages (on the Wayback Machine) from the Haiku Inc website. I've also updated the copyright notice at the bottom to 2020.

jt15s committed:
Create README.md

Added README.md, based on the README at the Haiku website repository, which provides documentation for contributers to the Haiku Inc website.

pulkomandy commented on content/donate/funded_development.md:
```diff
@@ -0,0 +1,153 @@
+# Development Contributions financed by Haiku, Inc.
```
This page is missing at least the following:

2019: Preetpal's work on Input preferences as part of Outreachy

2014-2016 (don't remember the exact dates):
- My work in 2014 on WebKit and general OS mprovements
- Pawel's work on the affine scheduler
- Barrett's work on media kit

Forgot the year:
- Waddlesplash's contract for the haikuports build infrastructure (it wasn't put in production, but as far as I know it was still funded)

2008 and 2009: "Haiku Code Drive", funding extra students aside GSoC when Google did not allow us as much students as we wanted to accept.

I can submit a later pull request adding these, when this one is merged.

jt15s replied:
@pulkomandy I think you're right, it's better to merge this first and then either you or I can get to finding all this other information for Contracts etc. 

Should we have a page describing GSOC and all the different years (i.e. who participated, related blog posts and what was done)?

pulkomandy replied:
The GSoC info is already at haiku-os.org/community/gsoc, there is no need to replicate it here. Maybe we can add a link to it, but it's a bit outside the scope here (since haiku inc didn't technically fund it, the payment was directly from Google to the students).

waddlesplash replied:
> Waddlesplash's contract for the haikuports build infrastructure (it wasn't put in production, but as far as I know it was still funded)

The contract was partially for the build infrastructure and partially just for "cleaning up HaikuPorts." While the Kitchen app never did get put into production, indeed, the haikuports cleanup obviously was merged (and part of that was migrating from Bitbucket to GitHub.)

pulkomandy commented on content/documents/requesting_funds.md:
```diff
@@ -0,0 +1,61 @@
+## Other Purchases
+In order to support conference and trade show presence, marketing supplies is another area that needs occasional funding. Hard goods, such as banners, presentation equipment, and fliers increase the visual appeal and effectiveness of Haiku at trade shows. Many of these items can be purchased and reused for several years, or purchased in bulk and distributed to several individuals attending differ ...
+
+Another group of expenses we would like to officially endorse, is purchasing items used for the development process. Examples would be hardware for writing new drivers or debugging existing ones, technical documents purchased from standards organizations, etc. We hope to also be able to provide developers with replacement hardware in the event of the inevitable failures that occur.
```
There is some hardware owned by Haiku inc that should be listed somewhere (either bought for developers, or donations from ahiku users and contributors or other non-profits):
- The Mac Minis from Mozilla
- Some ARM devboards (I think a gumstick currently in the hands of mmu_man, and possibly a beagleboard for a GSoC student, but we'd need to check
- Some VIA x86 CPU based embedded PC board with GPS receiver, Oxford Semi UART chip, etc (currently in my hands, but I can send it to other devs if they are interested)
- Anything else?

I think this deserves a new page and there could be a link here to show these as examples.

jt15s replied:
Hello @pulkomandy,

I've created a page paraphrasing your comment. Feel free to add to it once you've checked what hardware Haiku owns.

Thanks!

kallisti5 replied:
Some of the MacMini's have been "released" by the Inc. for their years of service, some are MIA, I have 4 of them sitting under my desk collecting dust. They didn't actually cost Haiku, Inc. anything, and were a generous donation from Mozilla (which we got good use out of for several years).  I've been pondering giving them to Haiku community members to develop Haiku on, but honestly a good chunk of them would just immediately go to EBay if I did that :-)

They're Core 2 Duo Mac's and honestly, running 4 of them is a waste of power as of a few years ago.  I took all 4 of mine offline since my personal Dell Core i7 system I have dedicated to nightly builds uses less energy and is faster.

jt15s replied:
@kallisti5 I've added a note after the Mac Minis saying that the Inc. released some to Haiku developers.

jt15s committed:
Added favicon and site header icons and updated documents section

I've added the Haiku Inc favicon, recreated in Inkscape to look like the original. I've added this to the site header as well. I've also updated the Documents section, adding a full PDF with all the pages for the IRS Certificate of Incorporation as well as fixing the broken link leading to the IRS filings.

jt15s committed:
Create hardware.md

Created hardware.md as suggested by @pulkomandy

waddlesplash commented on README.md:
```diff
@@ -0,0 +1,33 @@
+
+This repository contains the source code for [Haiku Inc's website](https://www.haiku-inc.org).
+
+## Adding new content
```
I don't know if we really need this information in this repo. The inc website is in a separate repository because it is supposed to be seldom modified and only by people who have a good reason to edit it. This looks like information more suited to the readme of just the main website repo.

jt15s replied:
Yeah, that is true, but I thought it might be a good idea to leave a note for whoever comes across this and decides to edit something in future.

jt15s replied:
@waddlesplash I felt it might be a good idea for whoever at Haiku decides to edit the website and finds this repo - there was no documentation for me when I got started moving the pages over from the old site.

waddlesplash replied:
I think it would be better to have this on the main website repo; the inc repo follows the same structure that does.

pulkomandy replied:
I think it's fine to have this documentation here, it is not worded as a call for contributors, more as documentation for people who need to do something with the website.

jt15s committed:
Added info on HCD

As suggested by @pulkomandy, information on the Haiku Code Drives in 2008 and 2009 has been added. Additionally I have also added a note saying that some of the donated Mac Minis were later released by the Inc. to some Haiku developers.

pulkomandy commented on content/donate/infrastructure.md:
```diff
@@ -0,0 +1,54 @@
+Some weeks before the release of Haiku R1/Alpha 1, a commitment was made to overhaul the infrastructure that provides key services to the Project. Leading up to this, there had been numerous service issues with the previous shared hosting plan and free source code repository. These issues ranged from unexplainable slowdowns to even occasional downtime. In a few words, the needs of the Project wer ...
+
+## Dedicated Server
+Through discussions on the [haiku-development mailing list](https://www.freelists.org/list/haiku-development "haiku-development mailing list"), it was decided to acquire an [EX4 dedicated server](https://web.archive.org/web/20161128085411/http://www.hetzner.de/en/hosting/produkte_rootserver/ex4/ "EX4 dedicated server") with Hetzner. This increased the hosting costs from roughly $300/year to $900/ ...
```
This info is out of date, the hosting has since been moved elsewhere

jt15s replied:
I've updated that, thanks!

pulkomandy commented on content/donate/infrastructure.md:
```diff
@@ -0,0 +1,54 @@
+	- [svn.haiku-os.org](https://svn.haiku-os.org "svn.haiku-os.org"): Old source code repository. Previosly mirrored back to [BerliOS](https://web.archive.org/web/20161128085411/http://developer.berlios.de/projects/haiku/ "BerliOS"), to provide additional exposure.
+	- [hg.haiku-os.org](https://hg.haiku-os.org "hg.haiku-os.org"): Read-only mirror of Haiku's repository. Miscellaneous source repositories (modifications to Trac).
+	- [server-stats.haiku-os.org](https://server-stats.haiku-os.org "server-stats.haiku-os.org"): A visual look into the resource usage into the dedicated server. Nearly every aspect of the key services (www, dev, svn) can be inspected via graph.
+	- [web-stats.haiku-os.org](https://web-stats.haiku-os.org "web-stats.haiku-os.org"): Usage statistics, specific to website traffic
```
svn, hg, server-stats and web-stats have been retired.

review.haiku-os.org, build.haiku-os.org, i18n.haiku-os.org have been added.

jt15s replied:
@pulkomandy I've fixed all of them. What software powers the i18n translation site? Is it Pootle?

pulkomandy replied:
There is Pootle for the user interface (however it is unmaintained, so we should switch to weblate or some other tool someday), and a custom tool for the userguide (called "Haiku userguide translator")

pulkomandy commented on content/donate/infrastructure.md:
```diff
@@ -0,0 +1,54 @@
+	- [api.haiku-os.org](https://api.haiku-os.org "api.haiku-os.org"): (Work-in-Progress) The Haiku Book
+
+## 3rd Party Sites
+In some cases, hosting is provided to third-parties who provide key services to the project.
```
This is not the case anymore, haikuports has moved to github.

jt15s replied:
Fixed. Thanks!

pulkomandy commented on content/donate/infrastructure.md:
```diff
@@ -0,0 +1,54 @@
+	- [ports.haiku-files.org](https://ports.haiku-files.org "ports.haiku-files.org"): Main site, Trac, SVN repository, and mailing lists.
+	- [ports-space.haiku-files.org](https://ports-space.haiku-files.org "ports-space.haiku-files.org"): Storage site for binary archives. SFTP upload and HTTP anonymous access.
+
+## Miscellaneous
```
I think this part can be removed now, haiku inc is no longer involved in these things

pulkomandy commented on content/donate/privacy_policy.md:
```diff
@@ -0,0 +1,12 @@
+# Donation Privacy Policy
+Your privacy will receive the utmost respect. The details of your donation will not be shared, without your express knowledge and consent. This includes your name, email address, donation amount, how many times you've donated, or if you are a monthly subscriber.
+
+Donors who have a yearly total of $50 USD, an equivalent EUR amount, or more may choose to have their name displayed on the Public Sponsors page. The intention is to provide additional incentive to people who appreciate such disclosure as a form of public recognition for their well-appreciated and valuable contributions to the Haiku Project. By default, your name will not be displayed.
```
we don't do this anymore, so I would omit this part unless we actually plan to do it again.

pulkomandy commented on content/trademarks/haiku_icons.md:
```diff
@@ -0,0 +1,17 @@
+# Haiku®'s Icons
```
The whole point of this page being that the icons are, in fact, not trademarked, it seems a bit useless?

jt15s replied:
@pulkomandy It seems this information is more suited to the Haiku Project website - maybe we can consider moving it there?

jt15s committed:
Updated pages

Updated pages to reflect the recent status of Haiku as suggested.

jt15s commented:
@leavengood Could you take a look at this PR and merge if it's ready? When I have time, I'll submit a PR adding more info into the "Funded Development" page when I have time. Thanks!

jt15s committed:
Merge remote-tracking branch 'upstream/master'

pulkomandy commented:
Hi,
It would be nice to get all this content back on the website so we can further update it. Anyone from the inc can handle that?

waddlesplash commented:
@kallisti5 or @leavengood should probably do so.

jt15s committed:
Merge remote-tracking branch 'upstream/master'

jt15s committed:
Update _index.md

Update info on Goodsearch & Goodshop, plus add info on Give with Bing

jt15s committed:
Update _index.md

Updated page to reflect the newer Google Pay name

jt15s committed:
Add Give with Bing info

Add info to remind users to select the right Give with Bing non-profit, as there are lots of non-profits on the program called "Haiku".

jt15s committed:
Update config.toml

Fixed Hugo build error

jt15s commented:
Okay, finally figured out what was causing the Netlify check to not pass

jt15s committed:
Fix page formatting (titles) and registered migrated pages in navigation menu

jt15s committed:
Create donations.md

Add placeholder page for future donation tracking page
